src/app/app.module.ts

```diff
@@ -25,10 +28,21 @@ import { HttpClientModule } from '@angular/common/http';
     HttpClientModule,
     EntitlementsModule,
     DropdownMenuModule,
-    AvatarModule,
     IconModule,
+    LayoutModule,
+    LogoModule,
+    NgbDropdownModule,
+    AvatarModule,
+    WebSdkModule.forRoot({
+      cx: { // TODO: this is a hack to get entitlements work using websdk
```

Comment:
Can you create a ticket to investigate how we can avoid this

Reply:
I know the reason. WebSDK is not supposed to handle entitlements like that. Those configurations are given by lean-client. But the hack was possible because WebSDK can send a whole configuration object to lean-client. The question here is should we add that functionality to foundation-ang?